test/unit_tests/modules/nav-primary-spec.js

```diff
@@ -0,0 +1,113 @@
+'use strict';
+var chai = require( 'chai' );
+var sinon = require( 'sinon' );
+var expect = chai.expect;
+var jsdom = require( 'mocha-jsdom' );
+
+describe( 'Get Event States', function() {
+  var $, navPrimary, sandbox;
+
+  jsdom();
+
+  before( function() {
+    $ = require( 'jquery' );
+    navPrimary = require( '../../../src/static/js/modules/nav-primary.js' );
+    sandbox = sinon.sandbox.create();
+  } );
+
+  beforeEach( function() {
+    //  Adding a simplified version of the thing we want to test.
+    //  Then calling the jQuery to test after
+    $( 'body' ).html(
```

Comment:
This is a test spec, does it matter? One of the things @imuchnik had said is we shouldn't use templates, just create the minimal markup for testing within the spec.